docs/options.md

```diff
@@ -449,9 +449,9 @@ Valid options:
 - `"auto"` – Format embedded code if Prettier can automatically identify it.
 - `"off"` - Never automatically format embedded code.
 
-| Default  | CLI Override                         | API Override                        |
-| -------- | ------------------------------------ | ----------------------------------- |
-| `"auto"` | `--embedded-language-formatting=off` | `embeddedLanguageFormatting: "off"` |
+| Default  | CLI Override                         | API Override                               |
+| -------- | ------------------------------------ | ------------------------------------------ |
+| `"auto"` | `--embedded-language-formatting=off` | `embeddedLanguageFormatting: "<auto|off>"` |
```

Comment:
Can you update `CLI Override` column too?

The `|` need escaped like other options(docusaurus bug).

Can you copy changes to /website/versioned_docs/version-stable/options.md file too, so the website get updated.

Reply:
Done